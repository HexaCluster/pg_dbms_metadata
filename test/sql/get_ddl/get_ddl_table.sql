SELECT dbms_metadata.get_ddl('TABLE','table_all','gdmmm');

SELECT dbms_metadata.get_ddl('TABLE','table_all_child','gdmmm');

SELECT dbms_metadata.get_ddl('TABLE','sales','gdmmm');
