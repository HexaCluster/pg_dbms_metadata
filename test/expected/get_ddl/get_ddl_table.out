                                                                                          get_ddl                                                                                           
--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 -- Table definition                                                                                                                                                                       +
 CREATE TABLE gdmmm.table_all (a integer DEFAULT nextval('gdmmm.table_all_a_seq'::regclass) NOT NULL,b integer  NOT NULL,c character varying(20) ,d integer ) TABLESPACE :"TABLESPACE_DATA"+
                                                                                                                                                                                           +
 -- Table comments                                                                                                                                                                         +
 COMMENT ON TABLE gdmmm.table_all IS 'This is a comment for the table.'                                                                                                                    +
                                                                                                                                                                                           +
 -- Column comments                                                                                                                                                                        +
 COMMENT ON COLUMN gdmmm.table_all.a IS 'This is a comment for the column.'                                                                                                                +
 COMMENT ON COLUMN gdmmm.table_all.b IS 'This is a comment for the column.'                                                                                                                +
 
(1 row)

                                                                          get_ddl                                                                          
-----------------------------------------------------------------------------------------------------------------------------------------------------------
 -- Table definition                                                                                                                                      +
 CREATE TABLE gdmmm.sales (sale_id integer  NOT NULL,sale_date date  NOT NULL,amount numeric ) TABLESPACE :"TABLESPACE_DATA" PARTITION BY RANGE(sale_date)
(1 row)

